build/Dockerfile

```diff
@@ -1,43 +1,35 @@
-FROM alpine:3.14.6@sha256:06b5d462c92fc39303e6363c65e074559f8d6b1363250027ed5053557e3398c5
+FROM debian:buster-slim@sha256:5b0b1a9a54651bbe9d4d3ee96bbda2b2a1da3d2fa198ddebbced46dfdca7f216
+
+# Setting bash as our shell, and enabling pipefail option
+SHELL ["/bin/bash", "-o", "pipefail", "-c"]
 
 # Some ENV variables
 ENV PATH="/mattermost/bin:${PATH}"
 ARG PUID=2000
 ARG PGID=2000
 ARG MM_PACKAGE="https://releases.mattermost.com/7.1.2/mattermost-7.1.2-linux-amd64.tar.gz?src=docker"
 
-
-# Install some needed packages
-RUN apk add --no-cache \
-  ca-certificates \
-  curl \
-  libc6-compat \
-  libffi-dev \
-  linux-headers \
-  mailcap \
-  netcat-openbsd \
-  xmlsec-dev \
-  tzdata \
-  wv \
-  poppler-utils \
-  tidyhtml \
-  && rm -rf /tmp/*
-
-# Get Mattermost
-RUN mkdir -p /mattermost/data /mattermost/plugins /mattermost/client/plugins \
-  && if [ ! -z "$MM_PACKAGE" ]; then curl $MM_PACKAGE | tar -xvz ; \
-  else echo "please set the MM_PACKAGE" ; fi \
-  && addgroup -g ${PGID} mattermost \
-  && adduser -D -u ${PUID} -G mattermost -h /mattermost -D mattermost \
-  && chown -R mattermost:mattermost /mattermost /mattermost/plugins /mattermost/client/plugins
-
+# # Install needed packages
+RUN apt-get update \
+  && apt-get install --no-install-recommends -y \
+  ca-certificates=20200601~deb10u2 \
+  curl=7.64.0-4+deb10u2 \
```

Comment:
This OS dependency list is  much smaller.  Are we sure everything needed in already included in the image?

Reply:
Thanks for commenting Doug

There is a query about exactly that posted @ https://community.mattermost.com/core/pl/88paijp4gjfpbykbrumej86noc

Any suggestions are more than welcome